UnitTests/Views/ListViewTests.cs

```diff
@@ -545,5 +545,24 @@ tem 2
  tem 3
  tem 4", output);
 		}
+
+		[Fact]
+		public void SelectedItemChanged_Event_Is_Also_Fired_With_AllowsMarking_True_By_Keyboard_Or_Mouse ()
```

Comment:
Very minor nitpick for consistency with the stack: The dotnet/c# term for events is "raise," not "fire."